File: app/src/main/java/id/co/gitsolution/androidchallege/Constant.java
package id.co.gitsolution.androidchallege;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

import id.co.gitsolution.androidchallege.model.Help;
import id.co.gitsolution.androidchallege.model.News;

/**
 * Created by Git Solution on 03/10/2017.
 */

public class Constant {
    public static List<News> getNews(){
        return new ArrayList<News>(Arrays.asList(
                new News("Content 1"),
                new News("Content 2"),
                new News("Content 3"),
                new News("Content 4")
        ));
    }

    public static List<Help> getHelp(){
        return new ArrayList<>(Arrays.asList(
                new Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017","189 Viewers"),
                new Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017","189 Viewers"),
                new Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017","189 Viewers"),
                new Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017","189 Viewers"),
                new Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017","189 Viewers")
        ));
    }
}

File: app/src/main/java/id/co/gitsolution/androidchallege/Constant.kt
package id.co.gitsolution.androidchallege

import id.co.gitsolution.androidchallege.model.Help
import id.co.gitsolution.androidchallege.model.News

/**
 * Created by Git Solution on 03/10/2017.
 */

object Constant {
    fun getNews(): List<News>{
        return listOf(
                News("My first experience with this was a year ago, trying to set more than 10 different fonts to the Android App I was developing at that moment"),
                News("My first experience with this was a year ago, trying to set more than 10 different fonts to the Android App I was developing at that moment"),
                News("My first experience with this was a year ago, trying to set more than 10 different fonts to the Android App I was developing at that moment"),
                News("My first experience with this was a year ago, trying to set more than 10 different fonts to the Android App I was developing at that moment"),
                News("My first experience with this was a year ago, trying to set more than 10 different fonts to the Android App I was developing at that moment")
        )
    }

    fun getHelp(): List<Help>{
        return listOf(
                Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017", "189 Viewers"),
                Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017", "189 Viewers"),
                Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017", "189 Viewers"),
                Help("Bagai mana cara nya untuk move on?", "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form", "23 Sept 2017", "189 Viewers")
        )
    }
}
